ec2/constants.py

```diff
@@ -47,3 +47,6 @@
 
 # keypair module constants
 KEYPAIR_REQUIRED_PROPERTIES = ['private_key_path']
+
+# Config
+AWS_CONFIG_PATH = '~/.aws_config.json'
```

Comment:
First, make this AWS_DEFAULT_CONFIG_PATH.

Then, fabric call in bootstrap will have an optional non-default path.

And utils will see if environment variable exists and read from it or fall back to the default path.

And we will rereview this